index.bs

```diff
@@ -7018,19 +7020,19 @@ Yuriy Ackermann,
 James Barclay,
 Richard Barnes,
 Dominic Battré,
-John Bradley,
+Julien Cayzac,
 Domenic Denicola,
 Rahul Ghosh,
 Brad Hill,
 Jing Jin,
 Wally Jones,
+Ki-Eun Shin,
```

Comment:
After Yaron Sheffer?

Reply:
I was going alphabetical by Family name.  
I believe in Korean the Family name is first, but this might be in western order given the number of syllables     @Kieun  can you confirm or reject my limited understanding.  
I will move to S for now.

Reply:
@ve7jtb thanks for adding. Confirmed.